integration-tests/pom.xml

```diff
@@ -17,7 +17,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-    </dependency>    
+  </dependency> 
```

Comment:
This looks like an accidental removal of a single space. 
Perhaps no need to check this in?

Reply:
yeah, oops, accident, presumably happened because I'd temporarily commented out hps-test-data dependency so it didn't redownload the data every time, fixed it